Failure to build space_robots due to missing dependency

I'm getting an error when trying to build the space_robots demo.

```
#17 161.7 --- stderr: ros_gz_bridge
#17 161.7 CMake Error at CMakeLists.txt:92 (find_package):
#17 161.7   By not providing "Findgps_msgs.cmake" in CMAKE_MODULE_PATH this project has
#17 161.7   asked CMake to find a package configuration file provided by "gps_msgs",
#17 161.7   but CMake did not find one.
#17 161.7
#17 161.7   Could not find a package configuration file provided by "gps_msgs" with any
#17 161.7   of the following names:
#17 161.7
#17 161.7     gps_msgsConfig.cmake
#17 161.7     gps_msgs-config.cmake
#17 161.7
#17 161.7   Add the installation prefix of "gps_msgs" to CMAKE_PREFIX_PATH or set
#17 161.7   "gps_msgs_DIR" to a directory containing one of the above files.  If
#17 161.7   "gps_msgs" provides a separate development package or SDK, be sure it has
#17 161.7   been installed.
#17 161.7
#17 161.7
#17 161.7 ---
#17 161.7 Failed   <<< ros_gz_bridge [11.6s, exited with code 1]
#17 162.1 Aborted  <<< ros_ign_interfaces [12.0s]
#17 175.9 Aborted  <<< velocity_controllers [1min 1s]
#17 177.8 Aborted  <<< position_controllers [58.5s]
#17 180.9 Aborted  <<< effort_controllers [1min 6s]
#17 231.1 Aborted  <<< qt_gui_cpp [2min 15s]
#17 234.1 Aborted  <<< vision_msgs_rviz_plugins [1min 54s]
#17 292.5 Aborted  <<< MONGO_CXX_DRIVER [3min 16s]
#17 292.6
#17 292.6 Summary: 24 packages finished [4min 51s]
#17 292.6   1 package failed: ros_gz_bridge
#17 292.6   7 packages aborted: MONGO_CXX_DRIVER effort_controllers position_controllers qt_gui_cpp ros_ign_interfaces velocity_controllers vision_msgs_rviz_plugins
#17 292.6   6 packages had stderr output: MONGO_CXX_DRIVER effort_controllers qt_gui_cpp ros_gz_bridge ros_ign_gazebo rqt_joint_trajectory_controller
#17 292.6   13 packages not processed
#17 ERROR: executor failed running [/bin/sh -c /bin/bash -c 'source ${SPACEROS_DIR}/install/setup.bash && source ${MOVEIT2_DIR}/install/setup.bash   && colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release']: exit code: 1
------
 > [stage-0 13/15] RUN /bin/bash -c 'source ${SPACEROS_DIR}/install/setup.bash && source ${MOVEIT2_DIR}/install/setup.bash   && colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release':
------
executor failed running [/bin/sh -c /bin/bash -c 'source ${SPACEROS_DIR}/install/setup.bash && source ${MOVEIT2_DIR}/install/setup.bash   && colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release']: exit code: 1
```

--- a/space_robots/demo_manual_pkgs.repos
+++ b/space_robots/demo_manual_pkgs.repos
@@ -35,3 +35,8 @@ repositories:
     type: git
     url: https://github.com/ros-perception/vision_msgs.git
     version: ros2
+  gps_msgs:
+    type: git
+    url: https://github.com/swri-robotics/gps_umd.git
+    path: gps_msgs
+    version: 113782d

